minerals/xyz.armcord.ArmCord.desktop

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
```

Comment:
Has this file been submitted upstream?

Reply:
Not yet, but I can do that. I'll make a MR and update the flatpak to use that when it's landed in a release.